projects/aws/bottlerocket-bootstrap/pkg/kubeadm/controlplane_init.go

```diff
@@ -73,8 +73,16 @@ func controlPlaneInit() error {
 		return errors.Wrap(err, "Error reading the ca data")
 	}
 
+	// Get local ApiServer port number from apiServer host string
+	port, err := getLocalApiServerBindPortFromInitConfig(kubeadmFile)
+	if err != nil {
+		// if we hit an error when extrating the port number, we always fallback to 6443
+		fmt.Printf("Unable to get local apiserver port, falling back to 6443")
```

Comment:
We're swallowing the error here - are we able to log that error so we understand why else it could be a simple pathing problem and we'll never know?

Reply:
Agree. Updating it to print out the error message.